spec/system/post/post_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe 'Index posts page', type: :system do
```

Comment:
- [ ] **[OPTIONAL]** I think file should be renamed `index_spec.rb` as that is more intuitive than `post_spec.rb`

Reply:
Excellent, I will rename that